Makefile

```diff
@@ -87,7 +91,6 @@ dc-build-clean:
 	rm -fr ./service-front/public/assets/v2/js/vendor; \
 	rm -fr ./service-front/vendor; \
 	rm -fr ./service-pdf/vendor; \
-	if [ "`docker network ls | grep malpadev`" = "" ] ; then docker network create malpadev ; fi; \
```

Comment:
Spotted these lines I missed in an earlier PR. These were to do with perfplat and are no longer needed so I cleaned them up while I was looking at the Makefile.